doc/squashfs-mount.1

```diff
@@ -0,0 +1,32 @@
+.TH SQUASHFS-MOUNT "1" "July 2024" "squashfs-mount 1.0.0" "User Commands"
+.SH NAME
+squashfs-mount \-  setuid utility instead of FUSE for mounting squashfs files
+.SH SYNOPSIS
+.B squashfs-mount
+\fI\,<image>:<mountpoint> \/\fR[\fI\,<image>:<mountpoint>\/\fR]...  \fI\,-- <command> \/\fR[\fI\,args\/\fR...]
+.SH DESCRIPTION
+This manual page documents briefly the
+.B squashfs-mount
+command.
+This manual page was written for the Debian distribution
```

Comment:
This part can be dropped IMHO (i.e. "This manual page was written for the Debian distribution because the original program does not have a manual page.")